app/scripts/modules/core/delivery/triggers/nextRun.component.js

```diff
@@ -2,7 +2,7 @@
 
 const angular = require('angular');
 
-let later = require('later');
+import later from 'core/utils/later/later';
```

Comment:
Since the file only had one import, it happened to work the way it was, but you are correct, it should be in curly braces.